server/http/helloapp/oauth.go

```diff
@@ -0,0 +1,128 @@
+package helloapp
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-api/experimental/bot/logger"
+	"github.com/mattermost/mattermost-plugin-api/experimental/oauther"
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/constants"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+)
+
+func (h *helloapp) InitOAuther() error {
+	oauth2Config, err := h.GetOAuthConfig()
+	if err != nil {
+		return err
+	}
+	h.OAuther = oauther.NewFromClient(h.apps.Mattermost,
+		*oauth2Config,
+		h.finishOAuth2Connect,
+		logger.NewNilLogger(),
```

Comment:
This was a quick implementation. We may want to put a proper logger here.

Reply:
We don't have a Logger yet. I have a branch where I have some changes to make it (loggers) work to my satisfaction, but it's not ready yet. I'll add a TODO